components/brave_wallet_ui/common/async/bridge.ts

```diff
@@ -4,13 +4,10 @@
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
 import WalletApiProxy from '../wallet_api_proxy'
+import getWalletPanelApiProxy from '../../panel/wallet_panel_api_proxy'
+import getWalletPageApiProxy from '../../page/wallet_page_api_proxy'
 
-export default async function getAPIProxy (): Promise<WalletApiProxy> {
-  let api
-  if (window.location.hostname === 'wallet-panel.top-chrome') {
-    api = await import('../../panel/wallet_panel_api_proxy')
-  } else {
-    api = await import('../../page/wallet_page_api_proxy')
-  }
-  return api.default()
+export default function getAPIProxy (): WalletApiProxy {
+  return window.location.hostname === 'wallet-panel.top-chrome' ?
```

Comment:
I think the new linter will want the `?` on a new line since this is a multi line ternary.

Reply:
fixed in https://github.com/brave/brave-core/pull/10982/commits/4668abb684d9091c4ff2163527378fadf6aa651f